.github/workflows/update-mlflow.yml

```
@@ -0,0 +1,35 @@
+name: Update mlflow tags
```

```suggestion
name: Update action versions
```

Let's change the workflow file name in a similar way since now it launches a script that updates all actions versions.